ssl/record/ssl3_record_tls13.c

```diff
@@ -21,7 +21,7 @@
  *   -1: if the record's AEAD-authenticator is invalid or, if sending,
  *       an internal error occurred.
  */
-int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int send)
+int tls13_enc(SSL *s, SSL3_RECORD *recs, size_t n_recs, int sending)
```

Comment:
Formally speaking answer to this should be "don't use --strict-warnings with gcc 4.6 [or suppress -Wshadow]". We can't and shouldn't appease every compiler bug, especially those that are non-fatal like one in question (generated code is correct after all) and actually fixed. But I'm leaving this to second reviewer...